### PR TITLE
Multiple Grammar fixes

### PR DESCRIPTION
**Pull request by arpitpandey0209**

### CODE REVIEW

**Comment by arpitpandey0209** [comment]
Done the change @sofietoft :+1:

File: Add-ons/UmbracoCourier/Architecture/DataResolvers.md [hunk 1]
```diff
@@ -156,4 +156,4 @@ Configuring which macro property types contains references to other nodes
 ### UsercontrolWrapper
 * **Full name:** `Umbraco.Courier.DataResolvers.UsercontrolWrapper`
 * **Triggers on:**  The UsercontrolWrapper Data types
-*If the data type has a .ascx file selected as render, the file is added as a resource to ensure it is trasfered with the data type.
+*If the data type has an .ascx file selected as render, the file is added as a resource to ensure it is trasferred with the data type.
```
[the remark this answers is not in the document]
Comment (arpitpandey0209) [reply Add-ons/UmbracoCourier/Architecture/DataResolvers.md]: Haha. Done that.

**Comment by sofietoft** [comment]
Wuhuw! Thanks for the fixes @arpitpandey0209 

Do you have an account on [Our](https://our.umbraco.com)? Then I can give you a contrib badge.

**Comment by arpitpandey0209** [comment]
> Do you have an account on [Our](https://our.umbraco.com)? Then I can give you a contrib badge.

Yes @sofietoft I have an Our account: https://our.umbraco.com/member/275721

**Comment by jmayntzhusen** [comment]
Hey @arpitpandey0209 

I gave you a contrib badge on Our now, thanks again for your help! 🎉😁

**Comment by nul800sebastiaan** [comment]
Hi there @arpitpandey0209,

First of all: **A big #H5YR** for contributing to Umbraco during Hacktoberfest! We are very thankful for the huge amount of PRs submitted, and all the amazing work you've been doing :pray:

If you haven't heard already we have had a record breaking amount of PR's in both the CMS and the Documentation repository. You can read all about the numbers in the [Umbraco Hacktoberfest Aftermath blog post](https://umbraco.com/blog/the-2018-hacktoberfest-aftermath/).

In the spirit of Hacktoberfest we've prepared some **exclusive Umbraco swag** for all our contributors - _including you_! 

[Receive your swag!](https://goo.gl/forms/smjQwEMDM5cUnz6x1) :point_left: Please follow this link to fill out and submit the form, **before November 21st 2018**.

Following this date we'll be sending out all the swag, which also means that it might not reach your doorstep before December, so please bear with us and be patient :pray:

The only thing left to say is **thank you so much for participating in Hacktoberfest**! We really appreciate the help!

Kind regards,
The Umbraco PR Team & The Documentation Curators

**Pull request merged**